google-cloud-firestore/src/test/java/com/google/cloud/firestore/TransactionTest.java

```diff
@@ -260,6 +373,55 @@ public String updateCallback(Transaction transaction) {
     assertEquals(commit(ByteString.copyFromUtf8("foo5")), requests.get(9));
   }
 
+  @Test
+  public void limitsRetriesWithFailureAsync() throws Exception {
```

Comment:
This test doesn't seem to add test coverage over the existing `limitsRetriesWithFailure()` test. If you agree, we could probably remove it. Same applies to `limitsRetriesWithSuccessAsync()`.

Reply:
tests removed